schema/data_type.go

```diff
@@ -15,25 +15,41 @@ const (
 	// IntegerType is our exported const representing integer data types
 	IntegerType = DataType("http://www.w3.org/2001/XMLSchema#integer")
 
+	// IntegerListType is our exported const representing a value that is a comma
+	// separated list of integer instances. Used for transport data values.
+	IntegerListType = DataType("http://purl.org/iot/vocab/thingful#ntegerList")
```

Comment:
Very minor: maybe remove the "Used for transport data values." from the comment since it can be used for any Thing type?